src/integration-tests/certificates/certificates.spec.js

```diff
@@ -0,0 +1,16 @@
+const commonHelper = require('@commonTests')
```

Comment:
How are we planning to execute certificate related scripts automatically here? @priyanka-TL  Can we add it in the circle ci config?